spec/integration/indirector/report/yaml.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+require 'puppet/transaction/report'
+require 'puppet/indirector/report/yaml'
+require 'puppet/indirector/indirector_testing/report/yaml'
+
+describe Puppet::Transaction::Report::Yaml do
+  describe '#save' do
+    let(:model) { Puppet::IndirectorTesting }
```

Comment:
Since we're testing a specific terminus/implementation for find/save/etc reports, I prefer using the actual model, `Puppet::Transaction::Report`.